src/models.c

```diff
@@ -49,6 +49,7 @@
 #include <stdlib.h>         // Required for: malloc(), free()
 #include <string.h>         // Required for: memcmp(), strlen()
 #include <math.h>           // Required for: sinf(), cosf(), sqrtf(), fabsf()
+#include <climits>
```

Comment:
This should be `<limits.h>`

Reply:
Actually no, this should be limits.h in all cases except for MSVC based compilers (cl.exe and clang-cl.exe) which require it to be climits.

Reply:
is it possible to remove this include and just assign some default values?

Reply:
> The `<climits>` wrapper of `<limits.h>` is only available in C++, not C. Also it might be worth noting that in C++ those headers are not guaranteed to contain any declarations specified in the C standard unless explicitly specified in C++ standard.

Reply:
Of course. Just thought it could be cleaner to have defines for readability of where the values come from. I will just enter the calculations, remove the include and write a comment of what the number is.

Reply:
Nice, thank you very much!